integration/mediation-tests/tests-platform/tests-rabbitmq/src/test/resources/testng.xml

```diff
@@ -60,7 +60,7 @@
 
     <test name="RabbitMQ-Inbound-Endpoint-test" preserve-order="true" verbose="2">
         <classes>
-            <class name="org.wso2.carbon.esb.rabbitmq.inbound.RabbitMQInboundTestCase"/>
```

Comment:
RabbitMQInboundTestCase is removed. Is it intentional ?

Reply:
no, fixed. Thanks :)